pkg/volume/csi/csi_attacher_test.go

```diff
@@ -206,9 +207,9 @@ func TestAttacherWithCSIDriver(t *testing.T) {
 	defer func() {
 		utilfeature.DefaultFeatureGate = originalFeatures
 	}()
-	err := utilfeature.DefaultFeatureGate.Set("CSISkipAttach=true")
```

Comment:
Can `SetFeatureGateDuringTest` be used instead? It does all the logic of setting and restoring feature gates before and after the test